docs/build_docs/source/tools/converter.rst

```diff
@@ -0,0 +1,85 @@
+|                             |              |                                                                |
+| ``--output_path``           |              |                                                                |
++-----------------------------+--------------+----------------------------------------------------------------+
+| ``-t``                      | Yes          | Path to the topology file in PDB format.                       |
+|                             |              |                                                                |
+| ``--topology``              |              |                                                                |
++-----------------------------+--------------+----------------------------------------------------------------+
+| ``-d``                      | Yes          | Delete the original trajectory after conversion.               |
+|                             |              | Default: False.                                                |
+| ``--delete``                |              |                                                                |
++-----------------------------+--------------+----------------------------------------------------------------+
+| ``-n``                      | Yes          | Number of processors to use. Default: 1.                       |
+|                             |              |                                                                |
+| ``--n_processors``          |              |                                                                |
++-----------------------------+--------------+----------------------------------------------------------------+
+| ``--trajectory_name``       | Yes          | Name of PELE's trajectory files, e.g.                          |
+|                             |              |                                                                |
+|                             |              | ``trajectory.pdb`` or ``trajectory.xtc``.                      |
++-----------------------------+--------------+----------------------------------------------------------------+
+| ``--verify``                | Yes          | Double-check the trajectories for consistency                  |
+|                             |              |                                                                |
+|                             |              | after conversion.*                                             |
++-----------------------------+--------------+----------------------------------------------------------------+
+| ``--dont_verify``           | Yes          | Skip verification.*                                            |
++-----------------------------+--------------+----------------------------------------------------------------+
+
+\* **Verification** involves checking if the original coordinates match the new ones. If they do, it will delete the original
```

Comment:
We should add that the default behaviour is to only verify consistency when original trajectories are deleted after conversion.